src/reducers/url-state.js

```diff
@@ -108,9 +112,22 @@ const selectedThread: Reducer<ThreadIndex | null> = (state = null, action) => {
     case 'ISOLATE_LOCAL_TRACK':
       // Only switch to non-null selected threads.
       return (action.selectedThreadIndex: ThreadIndex);
-    case 'PROFILE_PUBLISHED':
-      // This may no longer be valid because of PII sanitization.
-      return null;
+    case 'SANITIZE_PROFILE': {
+      const { oldThreadIndexToNew } = action;
+      if (state === null || !oldThreadIndexToNew) {
```

Comment:
Shouldn't we return the `state` if thread indexes were not modified(`!oldThreadIndexToNew`) but there is a state(`state !== null`)?

Reply:
Woops, nice catch!